libstuff/libstuff.cpp

```diff
@@ -2614,12 +2630,10 @@ int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int6
     }
 
     uint64_t elapsed = STimeNow() - startTime;
-    string sqlToLog = sql;
     if ((int64_t)elapsed > warnThreshold || (int64_t)elapsed > 10000) {
-        SRedactSensitiveValues(sqlToLog);
-
         // Avoid logging queries so long that we need dozens of lines to log them.
-        sqlToLog = sqlToLog.substr(0, 20000);
+        string sqlToLog = sql.substr(0, 20000);
```

Comment:
I don 't think we need it (we didn't have it before). If you're actually reading the 20k log line it will be pretty obvious if it just ends in the middle of a query, and frankly, nobody is reading that anyway.

Reply:
Okey doke